Synergism.css

```diff
@@ -1514,32 +1521,46 @@ nav li:hover {
 }
 
 #shop {
-	position: relative;
-	padding: 0;
 	text-align: center;
 }
 
-#actualWrapper {
-	display: grid;
-	grid-template-columns: 5% [cons] 20% 2% [shop] 60%;
+#shopWrapper {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
 }
 
-#actualShop {
-	display: grid;
-	grid-template-columns: repeat(8, minmax(70px, 105px));
-	margin-top: 10px;
-	width: calc(105px * 8);
-	border: 3px solid plum;
-	grid-column: shop 2/4;
-	grid-row: 1;
+#actualWrapper {
+	display: flex;
+	width: 100%;
+    justify-content: center;
 }
 
-#actualShop > div {
+#actualShopContainer {
 	display: flex;
 	flex-direction: column;
 	align-items: center;
-	margin-top: 5px;
-	margin-bottom: 5px;
+}
+
+#actualShop {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: flex-start;
+    padding-right: 5px;
+    margin-top: 10px;
+    width: calc(105px * 8);
+    border: 3px solid plum;
+}
+
+#actualShop > div {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-top: 5px;
+    margin-bottom: 5px;
+    margin-left: 5px;
+    width: 100px;
+    white-space: nowrap;
```

Comment:
```suggestion
```
Adding this causes the text to not wrap when it overflows:
![image](https://user-images.githubusercontent.com/42794878/115965488-80d12680-a4f7-11eb-87a4-8587b0760133.png)

Reply:
I've made this now apply only to the level displays rather than the purchase buttons, as that was the intended change